main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FeedContactValidator.java

```diff
@@ -0,0 +1,46 @@
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static org.mobilitydata.gtfsvalidator.notice.SeverityLevel.WARNING;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsFeedInfo;
+
+@GtfsValidator
+public class FeedContactValidator extends SingleEntityValidator<GtfsFeedInfo> {
+
+  @Override
+  public void validate(GtfsFeedInfo entity, NoticeContainer noticeContainer) {
+    if (!entity.hasFeedContactEmail() && !entity.hasFeedContactUrl()
+        || entity.feedContactEmail().isBlank() && entity.feedContactUrl().isBlank()) {
+      noticeContainer.addValidationNotice(
+          new FeedContactNotice(
+              entity.csvRowNumber(), entity.feedContactEmail(), entity.feedContactUrl()));
+    }
+  }
+
+  @GtfsValidationNotice(
+      severity = WARNING,
+      files = @GtfsValidationNotice.FileRefs(GtfsFeedInfo.class),
+      urls = {
+        @GtfsValidationNotice.UrlRef(
+            label = "Original Python validator implementation",
+            url = "https://github.com/google/transitfeed")
+      })
+  static class FeedContactNotice extends ValidationNotice {
+    /** The row number of the faulty record. */
+    private final int rowNumber;
+
+    private final String feedContactEmail;
+
+    private final String feedContactUrl;
```

Comment:
Please add Java documentation for these fields. This is why a test fails on the Github actions.